catalog/transformer/catalog-transformer-html/src/main/resources/templates/template.hbs

```diff
@@ -0,0 +1,136 @@
+{{!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ --}}
+<html>
+  <head>
+    <style>
+      nav li {
+        display: inline-block;
+      }
+      .metacard {
+        margin: 0 auto;
+        width: 1000px;
+      }
+      .metacard-controls {
+        cursor: pointer;
+        text-align: right;
+      }
+      .metacard-panel {
+        border-style: solid;
+        margin-top: 10px;
+        margin-bottom: 10px;
+      }
+      i {
+        border: solid black;
+        border-width: 0 3px 3px 0;
+        display: inline-block;
+        padding: 3px;
+      }
+      .up {
+        transform: rotate(-135deg);
+        -webkit-transform: rotate(-135deg);
+      }
+      .down {
+        transform: rotate(45deg);
+        -webkit-transform: rotate(45deg);
+      }
+      .metacard-categories {
+        border-top: solid;
+      }
+      .category-table {
+        border-collapse: collapse;
+        width: 100%;
+      }
+      .category-table td, th {
+        border: 1px solid #ddd;
+        padding: 8px;
+      }
+      .category-table tr:nth-child(even) {
+        background-color: #f2f2f2;
+      }
+      .category-table tr:hover {
+        background-color: #ddd;
+      }
+      .category-table th {
+        padding-top: 12px;
+        padding-bottom: 12px;
+        text-align: left;
+      }
+      .attribute-name {
+        width: 40% !important;
+      }
+    </style>
+  </head>
+  <body>
+
+    <ul class="control-panel">
+      <input id="empty-fields-checkbox" type="checkbox" onchange="toggleEmptyFields()">Hide empty fields<br>
+
+      <div class="metacard-controls">
+        <span class="expand-card" onclick="expandAll()">Expand All</span>
+        <span> | </span>
+        <span class="collapse-card" onclick="collapseAll()">Collapse All</span>
+      </div>
+    </ul>
+
+    {{#each this}}
+      {{> metacard}}
+    {{/each}}
+
+    <script>
+      function init() {
```

Comment:
Unnecessary function?

Reply:
Yessir it is. Originally I put this in there in case so that we could hide the javascript control menu in case we were going to reuse this for the RTF transformer but it's not necessary anymore